runbooks/patterns/for-loops/for-next-iteration.runbook.md

```
@@ -0,0 +1,26 @@
+    description: All three iterations pass via prompted mode
+    commands:
+      - rd run --prompted for-next-iteration.runbook.md
+      - rd pass
+      - rd pass
+      - rd pass
+    result: COMPLETE
+---
+
+# FOR Next Iteration
+
+## 1. Iterate items
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)